### DISCUSSION TITLE
Bug in tar_mermaid in latest version of targets

### DISCUSSION THREAD

**Discussion by scanisius** [ropensci/targets#1189] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

In the most recent version of targets, if `tar_mermaid` is called from within a targets pipeline, the pipeline fails with the error message *'argument "store" is missing, with no default'*. Calling the same function outside of the pipeline does not result in this error.

This issue does not occur with targets 1.3.0, but was introduced in a later version.

I believe the cause is https://github.com/ropensci/targets/blob/711506818a1f4c28aceef6aec0b34917e36079e9/R/tar_mermaid.R#L59 where `tar_assert_allow_meta("tar_mermaidd")` should be changed to `tar_assert_allow_meta("tar_mermaidd", store)`.

**Comment by wlandau** [comment]
Thanks, should be fixed in 4657a6f5355a8999273f05a093d693ed272649dd.
